pkg/kubelet/cm/dra/manager.go

```diff
@@ -158,6 +163,7 @@ func (m *ManagerImpl) PrepareResources(ctx context.Context, pod *v1.Pod) error {
 		logger.V(3).Info("Processing resource", "pod", klog.KObj(pod), "podClaim", podClaim.Name)
 		claimName, mustCheckOwner, err := resourceclaim.Name(pod, podClaim)
 		if err != nil {
+			metrics.DRAOperationsErrorsTotal.WithLabelValues(metricLabel).Inc()
```

Comment:
If you will add the error dimension to the duration metric, you can use that metric's base to query how many errors there are.